engine/crates/engine-v2/src/executor/introspection/resolver.rs

```diff
@@ -3,17 +3,17 @@ use std::cell::RefCell;
 use engine_value::ConstValue;
```

Comment:
Looks like a lot, but it's mostly the same thing. Just a different input that applies GraphQL resolution mechanism for fields.